dev-docs/bidders/newspassid.md

```diff
@@ -0,0 +1,75 @@
+---
+Module Name: NewspassId Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: techsupport@newspassid.com
+layout: bidder
+title: Newspass ID
+description: LMC Newspass ID Prebid JS Bidder Adapter
+biddercode: newspassid
+gdpr_supported: false
+gvl_id: none
+usp_supported: true
+coppa_supported: false
+schain_supported: true
+dchain_supported: false
+userIds: criteo, id5Id, tdid, identityLink, liveIntentId, parrableId, pubCommonId, lotamePanoramaId, sharedId, fabrickId
+media_types: banner
+safeframes_ok: true
+deals_supported: true
+floors_supported: false
+fpd_supported: false
+pbjs: true
+pbs: false
+prebid_member: false
+multiformat_supported: will-bid-on-any
+---
+
+# Description
+
+LMC Newspass ID Prebid JS Bidder Adapter that connects to the NewspassId demand source(s).
+
+The Newspass bid adapter supports Banner mediaTypes ONLY.
+This is intended for USA audiences only, and does not support GDPR
+ 
+# Test Parameters
```

Comment:
Please move your test parameters under the definitions of the the params themselves. Everyone else defines the params first.

Reply:
updated.